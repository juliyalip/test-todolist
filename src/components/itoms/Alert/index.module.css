.container {
  position: absolute;
  top: 0;
  right: 30%;
  text-align: center;
  width: 250px;
  height: 80px;
  background: orange;
  padding-top: 20px;
  font-weight: 500;
}/*# sourceMappingURL=index.module.css.map */