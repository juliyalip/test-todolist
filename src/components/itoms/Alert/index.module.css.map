{"version":3,"sources":["index.module.scss","index.module.css"],"names":[],"mappings":"AAAA;EACE,kBAAA;EACA,MAAA;EACA,UAAA;EACA,kBAAA;EACA,YAAA;EACA,YAAA;EACA,kBAAA;EACA,iBAAA;EACA,gBAAA;ACCF","file":"index.module.css"}