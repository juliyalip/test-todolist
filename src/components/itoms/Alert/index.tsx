import React from 'react';
import style from './index.module.scss'

const Alert = () =>{
    return(
        <div className={style.container}>
            The file is empty. Please, enter something. 
        </div>
    )
}
export default Alert