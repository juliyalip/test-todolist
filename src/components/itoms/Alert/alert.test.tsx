import Alert from './index';
import { render, screen } from '@testing-library/react';

describe('Alert component', () => {
  test('render correctly', () => {
    render(<Alert />);
    const textElement = screen.getByText(
      'The file is empty. Please, enter something.',
    );
    expect(textElement).toBeInTheDocument();
    expect(textElement).toMatchSnapshot()
  });
});
