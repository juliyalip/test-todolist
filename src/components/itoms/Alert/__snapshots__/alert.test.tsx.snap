// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component render correctly 1`] = `
<div
  class="container"
>
  The file is empty. Please, enter something.
</div>
`;
